apps/sms/views/shared/js/activity_handler.js

```diff
@@ -302,11 +302,11 @@ var ActivityHandler = {
           var titlePromise;
 
           if (Settings.hasSeveralSim() && message.iccId) {
-            var simName = Settings.getSimNameByIccId(message.iccId);
-            titlePromise = formatValue(
-              'dsds-notification-title-with-sim',
-              { sim: simName, sender: sender }
-            );
+            titlePromise = Utils.getSimNameByIccId(message.iccId)
```

Comment:
nit: maybe align something like this, so that ```.then``` and ```id``` don't have the same indentation level?
```
titlePromise = Utils.getSimNameByIccId(message.iccId).then(
  (simName) => ({
    id: 'dsds-notification-title-with-sim',
    args: { sim: simName, sender: sender }
  })
);
```